.env.example

```diff
@@ -5,3 +5,20 @@ TEST_DB_URL=TheTestDatabaseConnectionUrl__Format:postgres://user:pass@host:5432/
 DB_URL=TheStaggingDatabaseConnectionUrl__Format:postgres://user:pass@host:5432/dbname
 DB_URL=TheProductionDatabaseConnectionUrl__Format:postgres://user:pass@host:5432/dbname
 TOKEN_KEY=TheTokenKey
+
+
+
+FclientId=facebook__clientId
```

Comment:
@Ramadhan0 let's be explicit with the keys. Instead of `FclientId` or `TclientId`, let's go with `FacebookClientId` and `TwitterClientId` etc.